### ISSUE TITLE
Request: Add different option for "System Language" and "Locales"

### ISSUE THREAD

**Issue by ThanosApostolou** [Antergos/Cnchi#779]
### Problem:
Request for feature to add different option for "System Language" and "Locales"

### Steps To Reproduce:
1. Try to do an installation using cnchi, while you need different system language and different locale settings.

### Log Files
- None

### Additional Notes:
Many people (most of people, in my opinion) need different language for their system and different locales. For example I use Greek locale settings, but I always have English for system language since it's easier to find terminology on the internet, when you search something and want to do advanced tasks.

It would be really nice to add 2 options, one for system's language and one for the rest locale settings. For example if I need English for system language but Greek for locales then the **/etc/locale.conf** should contain the values

```
LANG=en_US.UTF-8
LC_NUMERIC=el_GR.UTF-8
LC_TIME=el_GR.UTF-8
LC_MONETARY=el_GR.UTF-8
LC_PAPER=el_GR.UTF-8
LC_MEASUREMENT=el_GR.UTF-8
```

There are some other values like **LC_TELEPHONE** but I don't believe they are really useful nowadays. The above LC options are those which change by default, when someone changes the **"Formats"** at **"Reginon & Language"** from gnome settings. Do you think that you can implement this at the cnchi installer?

**Comment by Lantaros** [comment]
Yeah agree, I'm portuguese, so I use the pt-pt keyboard layout but I prefer to choose English as my system language. Using Cnchi and selecting those options. at the end I get everything in portuguese, even my home folder directories. For instance my "Desktop folder" is called "Área de Trabalho".
I would like to try to solve this If you were to set me on the right direction.
Thanks for your time
